ui/pages/confirmations/components/confirm/info/utils.test.ts

```diff
@@ -36,3 +41,103 @@ describe('getIsRevokeSetApprovalForAll', () => {
     expect(actual).toEqual(true);
   });
 });
+
+describe('hasValueAndNativeBalanceMismatch', () => {
```

Comment:
NitL if method is simulation specific and places in utils, may be we name it as something `simulationHasValueAndNativeBalanceMismatch`